patches/cleanUpEduVulcanHome.css

```diff
@@ -0,0 +1,8 @@
+/* eduVulcan features, app downlaod links, eduvulcan banner, maziaj */
+.part-constant:has(.carousel-area-no-authenticated), .box-app, .carousel-component, .ribbon-bg {
+    display: none !important;
```

Comment:
![opera_SbFdD261ES](https://github.com/user-attachments/assets/4f9b98b2-d4e1-4c36-855d-f6ade20e3a92)

Here you removes the part that gives me access to e-register

Reply:
It hides only for .part-constant**:has(.carousel-area-no-authenticated)**. Can you show HTML?